frontend/src/app/Enrollment/index.jsx

```diff
@@ -0,0 +1,175 @@
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import { useDispatch, useSelector } from 'react-redux';
+import ClassCardList from '../../components/ClassCards/ClassCardList';
+import EnrollmentGraphCard from '../../components/GraphCard/EnrollmentGraphCard';
+import EnrollmentSearchBar from '../../components/ClassSearchBar/EnrollmentSearchBar';
+
+import info from '../../assets/img/images/graphs/info.svg';
+
+import {
+	fetchEnrollContext,
+	fetchEnrollClass,
+	enrollRemoveCourse,
+	enrollReset,
+	fetchEnrollFromUrl
+} from '../../redux/actions';
+import { useCallback, useEffect, useState } from 'react';
+
+const toUrlForm = (s) => {
+	return s.toLowerCase().split(' ').join('-');
+};
+
+export function Component() {
```

Comment:
Understandable. There's probably a better way, but the `lazy` prop for routes requires a named export `Component`. See more: https://reactrouter.com/en/main/route/lazy. We can use `Component.displayName` to set the name of the component in React dev tools.

Reply:
Oh wow I didn't know about that requirement of `react-router` that is extremely strange of them to force for lazy loading. It could be a worthy sacrifice, but we could look into alternative lazy loading methods so we don't have to compromise.